docs/user-guide/persistent-volumes/index.md

```diff
@@ -216,3 +231,122 @@ spec:
 ### A Note on Namespaces
 
 `PersistentVolumes` binds are exclusive, and since `PersistentVolumeClaims` are namespaced objects, mounting claims with "Many" modes (`ROX`, `RWX`) is only possible within one namespace.
+
+## StorageClasses
+
+Each `StorageClass` contains the fields `provisioner` and `parameters`, which 
+are used when a `PersistentVolume` belonging to a class needs to be dynamically
+ provisioned.
+
+The name of a `StorageClass` object is significant, and is how users can 
+request a particular class. Administrators set the name and other parameters 
+of a class, all of which are opaque to users, when first creating 
+`StorageClass` objects, and the objects cannot be updated once they are 
+created.
+
+Users don't necessarily have to request a `StorageClass` in their PVC. The
+cluster treats PVCs that don't request a `StorageClass` differently depending
+on whether the
+[`SimpleDefaultStorageClassForPVC` admission controller](docs/admin/admission-controllers/#simpledefaultstorageclassforpvc) is turned on.
+* If the admission controller is turned on, the administrator may specify a
+default `StorageClass`. All PVCs that don't request a `StorageClass` will be
+bound only to PVs of that default. Specifying a default `StorageClass` is done
+by setting the annotation `storageclass.beta.kubernetes.io/is-default-class`
+equal to "true" in a `StorageClass` object. If the administrator does not
+specify a default, the cluster will respond to PVC creation as if the admission
+controller were turned off. If more than one default is specified, the
+admission controller will forbid the creation of all PVCs.
+* If the admission controller is turned off, the claim may be bound to any
+volume that does not belong to a `StorageClass` or that has a blank `""` value
+for its `volume.beta.kubernetes.io/storage-class` annotation.
+
+```yaml
+kind: StorageClass
+apiVersion: extensions/v1beta1
+metadata:
+  name: standard
+provisioner: kubernetes.io/aws-ebs
+parameters:
+  type: gp2
+```
+
+### Provisioner
+Storage classes have a provisioner that determines what volume plugin is used 
+for provisioning PVs. This field must be specified. During beta, the available
+provisioner types are `kubernetes.io/aws-ebs` and `kubernetes.io/gce-pd`.
+
+### Parameters
+Storage classes have parameters that describe volumes belonging to the storage 
+class. Different parameters may be accepted depending on the `provisioner`. For
+ example, the value `io1`, for the parameter `type`, and the parameter 
+`iopsPerGB` are specific to EBS. When a parameter is omitted, some default is 
+used.
+
+#### AWS
+
+```yaml
+kind: StorageClass
+apiVersion: extensions/v1beta1
+metadata:
+  name: slow
+provisioner: kubernetes.io/aws-ebs
+parameters:
+  type: io1
+  zone: us-east-1d
+  iopsPerGB: "10"
+```
+
+* `type`: `io1`, `gp2`, `sc1`, `st1`. See AWS docs for details. Default: `gp2`. 
+* `zone`: AWS zone. If not specified, a random zone in the same region as 
```

Comment:
actually, it's not "random zone in the same region", it's "a random zone where this Kubernetes cluster has a node". I'll update kubernetes/examples/experimental/persistent-volume-provisioning.